### PR TITLE
Process for recognizing work

### PR DESCRIPTION
**Pull request by lirantal**
References:
* https://github.com/nodejs/security-wg/issues/293 - Acknowledging most active hackers for ecosystem #293 
* https://github.com/nodejs/security-wg/issues/311 - Schedule Tweet - Agree on text #311

cc @ZibbyKeaton for review

### CODE REVIEW

File: processes/recognition.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Recognition
+
+Outlining the processes, tools and resources available for recognizing the work being done in the space of the Security WG security researchers.
```
Comment (vdeturckheim) [inline processes/recognition.md]: I would add a mention here stating this is about third parties.

**Comment by vdeturckheim** [comment]
Good first version. I don't have much to say regarding format/content so far

File: processes/recognition.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### Swag TBD
+
+* Stickers
+* T-shirts
+* Books
+
+### Recognition for Significant Contribution TBD
+
+Defining a process/threshold for what constitutes a significant contribution and what it yields (recognition artifacts, joining the wg, etc)
```
Comment (mcollina) [inline processes/recognition.md]: I think joining the WG would be a good way to attract new members!

Comment (lirantal) [reply processes/recognition.md]: 👍

**Comment by lirantal** [comment]
Updated TBD sections to make them clear about it and removed the agenda since we agreed to land this as is for now.

**Pull request merged**